ui/app/templates/workspace/projects/new.hbs

```diff
@@ -28,7 +28,7 @@
     <fieldset class="pds-formFieldSet">
       <div class="pds-formField">
         <div class="pds-fieldName" id="create-git-help-text">
-          <Pds::Icon @type="git-repository" class="icon" />
+          <FlightIcon @name="git-repo" class="icon" />
```

Comment:
Was this icon meant to grow?

<img width="415" alt="CleanShot 2021-12-09 at 10 18 05@2x" src="https://user-images.githubusercontent.com/34030/145377996-cf399bc4-e2e7-42dc-a661-a99954c95fc8.png">